pkg/filter/mirror-config/v1alpha1/mirror-filter.go

```diff
@@ -389,3 +399,34 @@ func bundleNames(bundles []SelectedBundle) []string {
 	}
 	return bundleNames
 }
+
+// isSpecificVersion checks if a version range represents a specific version match
+// (i.e., min version == max version). This is true for version ranges like ">=26.0.8-opr.1 <=26.0.8-opr.1".
+func isSpecificVersion(versionRange string) (bool, string) {
+	parts := strings.Fields(versionRange)
+
+	if len(parts) == 2 {
+		minVer := strings.TrimPrefix(parts[0], ">=")
+		maxVer := strings.TrimPrefix(parts[1], "<=")
+
+		if minVer == maxVer {
+			return true, minVer
+		}
+	}
+
+	return false, ""
+}
+
+// specificBundle finds the bundle with the specific version in the bundles map and returns the bundle name
+func specificBundle(specificVersion string, bundles map[string]*mmsemver.Version) (string, error) {
+	for bundle, bundleVersion := range bundles {
+		specificVersion, err := mmsemver.NewVersion(specificVersion)
+		if err != nil {
+			return "", fmt.Errorf("error parsing specific version: %w", err)
+		}
```

Comment:
The conversion `string` -> `mmsemver.Version` can be done once before the loop:
```suggestion
    specificVersion, err := mmsemver.NewVersion(specificVersion)
	if err != nil {
		return "", fmt.Errorf("error parsing specific version: %w", err)
	}
	for bundle, bundleVersion := range bundles {
```